src/sentry/static/sentry/app/components/modals/sentryAppDetailsModal.jsx

```diff
@@ -0,0 +1,92 @@
+import {Box, Flex} from 'grid-emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import Access from 'app/components/acl/access';
+import Button from 'app/components/button';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+
+export default class SentryAppDetailsModal extends React.Component {
+  static propTypes = {
+    sentryApp: SentryTypes.SentryApplication.isRequired,
+    organization: SentryTypes.Organization.isRequired,
+    onInstall: PropTypes.func.isRequired,
+    isInstalled: PropTypes.bool.isRequired,
+    closeModal: PropTypes.func.isRequired,
+  };
+
+  render() {
+    const {sentryApp, closeModal, onInstall, isInstalled, organization} = this.props;
+
+    return (
+      <React.Fragment>
+        <Flex align="center" mb={2}>
```

Comment:
I'm assuming this is copy-pasta from the other modals, but it seems weird that we have some inline styling and then some using the styled components. I feel like they should all be styled components